tests/conftest.py

```diff
@@ -29,6 +29,7 @@ def pytest_addoption(parser):
     parser.addoption("--disable_sql_simplifier", action="store_true", default=False)
     parser.addoption("--local_testing_mode", action="store_true", default=False)
     parser.addoption("--enable_cte_optimization", action="store_true", default=False)
+    parser.addoption("--multithreading_mode", action="store_true", default=False)
```

Comment:
now since you have a parameter now, the multithreading_mode can probably work in a very similar way as other parameters